orc8r/cloud/go/services/state/indexer/indexer.go

```
@@ -68,6 +69,9 @@ type Version uint32
 // NewIndexerVersion returns a new indexer version, ensuring it fits into
 // the required integer size.
 func NewIndexerVersion(version int64) (Version, error) {
+	if version < 0 || version > math.MaxUint32 {
+		return 0, fmt.Errorf("version value %d is outside the boundaries of uint32 type", version)
+	}
 	capped := Version(version)
 	if int64(capped) != version {
 		return 0, fmt.Errorf("indexer version %v too large for %T", version, Version(0))
```

This check is doing the same thing as the check below it. I'm fine with the "new" way if it silences an error, but then let's remove the old way

---

I removed the old way...